core/src/main/java/com/scalar/db/api/DistributedTransactionAdmin.java

```diff
@@ -96,6 +96,14 @@ default void dropCoordinatorTables(boolean ifExist) throws ExecutionException {
    */
   boolean coordinatorTablesExist() throws ExecutionException;
 
+  /**
+   * Repair coordinator tables which may be in an unknown state
+   *
+   * @param options options to repair
+   * @throws ExecutionException if the operation failed
+   */
+  void repairCoordinatorTables(Map<String, String> options) throws ExecutionException;
+
```

Comment:
The definition of the new `repairCoordinatorTables` available on transactional admin interfaces.